test/app-luatest/gh_4317_console_backslash_test.lua

```diff
@@ -0,0 +1,57 @@
+local t = require('luatest')
+local g = t.group()
+
+local result_str =  [[tarantool> \set c \
+---
+- true
+...
+
+tarantool> local a = 0 \
+         > for i = 1, 10 do
+         > a = a + i
+         > end \
+         > print(a)
+55
+---
+...
+
+tarantool> ]]
+
+local TARANTOOL_PATH = arg[-1]
```

Comment:
Strictly saying, this is wrong: the binary is placed to the *least* negative index (consider the case if CLI flags are used). You can find the recipe [here](https://github.com/tarantool/luajit/blob/tarantool/test/lua-Harness-tests/test_assertion.lua#L14-L20).

Reply:
Hm. `arg[-1]` is widely used in our tests. I don't see how CLI flags can move the executable name/path to `arg[-2]` or so. Can you show an example how to achieve it?

Reply:
Created an issue for this (#7936).

Reply:
Okay, let's discuss it in the separate issue. I have added my information.

Reply:
It seems, `arg[-1]` just works for tarantool and we need to bother only if we'll decide to follow PUC-Rio Lua 5.1 as result of #7936 discussion (I hope we'll not). I vote for removing the comment, but I don't insist. It is up to Gleb and Igor.

Reply:
We decided to leave it as is for now.

Reply:
Since there is an agreement, I'll resolve the discussion.